test/TestConfig/extraSettings.mk

```diff
@@ -71,7 +71,7 @@ ifneq (,$(findstring win,$(SPEC)))
 else
 	JAVA_SHARED_LIBRARIES_DIR:=$(JAVA_LIB_DIR)$(D)$(ARCH_DIR)$(D)$(VM_SUBDIR)
```

Comment:
This definition is not appropriate for Java 9 (the arch directory was removed ~b150).

Reply:
A better solution is to move the test shared libraries out of the VM.

Reply:
i.e. #805

Reply:
That said, I don't see why we can't merge this change and leave #805 for another day. This comment doesn't seem to have anything to do with the additional to the LIBPATH below. I also note that while JAVA_SHARED_LIBRARIES_DIR appears wrong for Java 9, the testing is working as expected.

Reply:
Or to put this another way, I expect this setting for JAVA_SHARED_LIBRARIES_DIR supports the IBM internal b148 build. If we're going to update it, we'll need to do something internal to IBM in order to keep b148 tests working.